### PR TITLE
Create AngularJS Styleguide

### PR DESCRIPTION
**Pull request by jdalt**
I'd like to create a style guide page that includes our specific style choices and idioms for AngularJS. 

We should probably base most of our style choices off of a larger community accepted guide (https://github.com/johnpapa/angular-styleguide, https://github.com/mgechev/angularjs-style-guide, https://github.com/toddmotto/angular-styleguide, https://google.github.io/styleguide/angularjs-google-style.html). 

Personally I'd like to hew to the community consensus for doing angular as closely as possible and then use this guide to resolve any lingering ambiguity. The weirder we, get the less things like StackOverflow will be there to help us.

### CODE REVIEW

File: angular.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+layout: page
+title: Angular
+header: Angular Style Guide
+group: navigation
+---
+{% include JB/setup %}
+
+This is a list of style choices and idioms we wish to apply when using AngularJs.
+
+## Acces to Parent Scope
```
Comment (ScottTrenda) [inline angular.md]: access

File: angular.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+When a parameter is assigned in a template, it will sometimes need acess to the parent scope:
```
Comment (ScottTrenda) [inline angular.md]: access

Comment (ScottTrenda) [inline angular.md]: might want to say "explicit access" here, and explain why

**Comment by ScottTrenda** [comment]
CR :+1:

**Comment by jdalt** [comment]
:clap:

**Pull request merged**